addons/web/static/tests/views/search_panel_tests.js

```diff
@@ -1381,7 +1382,7 @@ QUnit.module('Views', {
                 'partner,false,search': `
                     <search>
                         <searchpanel>
-                            <field name="state"/>
+                            <field name="state" disable_counters="1"/>
```

Comment:
have a similar test with a m2o if not yet

Reply:
done